### PR TITLE
/openstack/install copy update

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit)
- Ran djlint

## QA

- View the site locally in your web browser at: https://ubuntu-com-13837.demos.haus/openstack/install
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10910

### CODE REVIEW

File: templates/openstack/install.html [hunk 1]
```diff
@@ -1,164 +1,226 @@
...
-        hi_def=True,
-        loading="auto",
-        ) | safe
-      }}
+  <section class="p-strip--suru-bottomed is-bordered">
+    <div class="row u-equal-height u-vertically-center spaced-bottom">
+      <div class="col-7">
+        <h1>Install OpenStack yourself</h1>
+        <p>
+          Try OpenStack in five simple steps. Set up a small-scale cloud or deploy a large cluster across hundreds of physical machines. Use <a href="https://microstack.run/">MicroStack</a> or <a href="/openstack">Charmed OpenStack</a> depending on your use case.
```
Comment (mtruj013) [inline templates/openstack/install.html:19]: Change here

File: templates/openstack/install.html [hunk 2]
```diff
@@ -1,164 +1,226 @@
...
+           role="tabpanel"
+           id="single-node-mode"
+           aria-labelledby="single-node-mode-tab">
+    <div class="p-strip is-shallow u-no-padding--top">
+      <div class="row">
+        <h2>Single-node mode</h2>
+        <div class="col-6">
+          <ul class="p-list">
+            <li class="p-list__item is-ticked">Uses MicroStack</li>
+            <li class="p-list__item is-ticked">Works on machines with limited hardware resources</li>
```
Comment (mtruj013) [inline templates/openstack/install.html:94]: Change here

File: templates/openstack/install.html [hunk 3]
```diff
@@ -1,164 +1,226 @@
...
-        <p class="p-card__content">Learn OpenStack through a series of tutorials. Starting with just your workstation, learn how to use OpenStack for cloud infrastructure implementation purposes, from a single-node installation to large-scale clusters.
+        <img class="p-card__thumbnail--large"
+             src="https://assets.ubuntu.com/v1/3a4d4155-Training.svg"
+             alt="" />
+        <hr class="u-sv1" />
+        <p>
+          <a href="/openstack/tutorials">Check our tutorials&nbsp;&rsaquo;</a>
+        </p>
+        <p class="p-card__content">
+          Learn OpenStack through a series of tutorials. Starting with just a single machine, learn how to use OpenStack for cloud infrastructure implementation purposes, from a single-node installation to large-scale clusters.
```
Comment (mtruj013) [inline templates/openstack/install.html:171]: Change here

**Review by minkyngkm — APPROVED** [review]
LGTM 👍

**Pull request merged**